webui/react/src/e2e/models/pages/WorkspaceList/WorkspaceCard.ts

```diff
@@ -0,0 +1,12 @@
+import { Card } from 'e2e/models/common/hew/Card';
+import { WorkspaceActionDropdown } from 'e2e/models/components/WorkspaceActionDropdown';
+
+/**
+ * Represents the WorkspaceCard src/pages/WorkspaceList/WorkspaceCard.tsx
```

Comment:
this previously existed on workspace list, but it was actually it's own file in the source